ui/app/routes/vault/cluster/secrets/backend/list.js

```diff
@@ -62,11 +64,16 @@ export default Route.extend({
     const { tab } = this.paramsFor('vault.cluster.secrets.backend.list-root');
     const secretEngine = this.store.peekRecord('secret-engine', backend);
     const type = secretEngine && secretEngine.get('engineType');
+    const { engineRoute } = allEngines().findBy('type', type);
+
     if (!type || !SUPPORTED_BACKENDS.includes(type)) {
-      return this.transitionTo('vault.cluster.secrets');
+      return this.router.transitionTo('vault.cluster.secrets');
```

Comment:
There was a deprecation warning for using router methods directly on the route instance which is why I updated these to use the router service.